aip/general/0203.md

```diff
@@ -142,20 +148,6 @@ Potential use cases for immutable fields (this is not an exhaustive list) are:
 **Note:** Fields which are "conditionally immutable" **must not** be given the
 immutable annotation.
 
-### Optional
-
-The use of `OPTIONAL` indicates that a field is not required, nor covered by
```

Comment:
The reason that "not covered by any of the above descriptions" was included is because when the annotation was being debated, API producers wanted to avoid the outcome of having to specify `field_behavior` on every field. We almost didn't provide `OPTIONAL` at all, but Docgen wanted it for cases where documentation needed to call out that the field was optional (e.g. if it was counter-intuitive).

Reply:
Great, that matches my understanding as well. There's been a lot of discussion internally around this and the conclusion (with ATL review) was that we should require these, and the cost to API producers is worth the benefit to clients.